app/src/main/java/one/mixin/android/job/SendTranscriptAttachmentMessageJob.kt

```diff
@@ -118,9 +119,13 @@ class SendTranscriptAttachmentMessageJob(
             },
             {
                 Timber.e("upload attachment error, ${it.getStackTraceString()}")
-                reportException(it)
-                removeJob()
-                transcriptMessageDao.updateMediaStatus(transcriptMessage.transcriptId, transcriptMessage.messageId, MediaStatus.CANCELED.name)
+                if (it.worthRetrying()) {
+                    throw it
```

Comment:
It is better to use suspend function or synchronization function [here](https://github.com/MixinNetwork/android-app/blob/7852a142520fae243cad3434788479c613156b9c/app/src/main/java/one/mixin/android/api/service/ConversationService.kt#L32).